fava/api/__init__.py

```diff
@@ -281,7 +285,7 @@ def _real_account(self, account_name, entries, begin_date=None,
     def balances(self, account_name, begin_date=None, end_date=None):
         real_account = self._real_account(account_name, self.entries,
                                           begin_date, end_date)
-        return [serialize_real_account(real_account)]
+        return [serialize_real_account(real_account, self.budgets.budget)]
```

Comment:
When doing this the child-accounts won't have any budget information. By passing the function here it can be called for each of the children in `serialize_real_account`.